### ISSUE TITLE
Cant adjust Stroke width

### ISSUE THREAD

**Issue by jellybytes** [rejuvenate/lovelace-horizon-card#116]
Is there a setting to set the stroke width of the trajectory or axis in the yml config? Also searched the js file, but cant find a variable to tweak it myself. Any help appreciated ;D Great card btw!

**Comment by FoxMike1** [comment]
Yes please. I need that to, in dark mode I hardly can see the thin trajectory lines. It woul be great to mod them in size and colour just like the names or values of the card.

**Comment by hheimbuerger** [comment]
I would like this, too. I'm displaying the widget on an early-generation grayscale eInk display (Kindle 4), and 1-pixel wide lines are almost invisible.

**Comment by ThomDietrich** [comment]
Thanks for the patience on this one. There is no dedicated config option, but every line in the graph now carries a stable CSS class, so you can set the width yourself with Uix or card-mod:

```yaml
uix:   # or card_mod:
  style: |
    .horizon-card-sun-path,
    .horizon-card-horizon-line,
    .horizon-card-sunrise-line,
    .horizon-card-sunset-line {
      stroke-width: 2px;
    }
```

The width is in the graph's viewBox units (it is 550 wide) and the default is `1`, so `2px` is roughly double. A plain theme file can only set the colours (the `--hc-*` variables), not the width, so line thickness needs Uix or card-mod.

@FoxMike1 @hheimbuerger this should help with the dark-mode and e-ink readability you both mentioned.

I have added this and a few other common tweaks to the README styling section in #232. Closing as covered, but shout if a first-class option or an `--hc-line-width` variable would still be useful. Cheers!
